public/app/plugins/datasource/testdata/components/SimulationSchemaForm.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FormEvent } from 'react';
+
+import { DataFrameSchema } from '@grafana/data';
+import { InlineField, Input, FieldSet } from '@grafana/ui';
+interface SchemaFormProps {
+  config: Record<string, any>;
+  schema: DataFrameSchema;
+  onChange: (config: Record<string, any>) => void;
+}
+
+export const SimulationSchemaForm = ({ config, schema, onChange }: SchemaFormProps) => {
+  return (
+    <FieldSet>
+      {schema.fields.map((field) => (
+        <InlineField label={field.name} key={field.name}>
```

Comment:
rather than if else... we could do a switch statement.  

Can we set a consistent label width?